Resources/doc/filter/sort.rst

```diff
@@ -0,0 +1,124 @@
+===========
+Sort filter
+===========
+
+Filter used for sorting the documents.
+
+For example, lets say we have `item` repository which contains the following data:
+
++---------+------------+
+| item_id | item_color |
++=========+============+
+| 1       | red        |
++---------+------------+
+| 2       | red        |
++---------+------------+
+| 3       | blue       |
++---------+------------+
+| 4       | green      |
++---------+------------+
+| 5       | blue       |
++---------+------------+
+
+If we have a sort filter with choice to sort on `item_color` in descending order and this choice is selected we will get:
```

Comment:
If we have a sort filter with choice to sort on `item_color` in descending order and this choice is selected, we will get: